### PR TITLE
Kubeflow documentation

### PR DESCRIPTION
**Pull request by foxish**
Did a broad pass at the documentation. Lots of TODOs. PTAL at the overall structure as well.

cc/ @vishh @jlewi @kow3ns @aronchick

### CODE REVIEW

File: kubeflow/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Kubeflow Anywhere
+
+This document detals the steps needed to run the kubeflow project in different environments, such as Minikube (local laptop). Google Kubernetes Engine, etc.
```
Comment (jlewi) [inline kubeflow/README.md]: details

File: kubeflow/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Kubeflow Anywhere
```
Comment (jlewi) [inline kubeflow/README.md]: Should there be a brief intro/overview explaining what Kubeflow is?

File: kubeflow/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+In order to quickly setup all components of the stack, run:
+
+```commandline
+kubectl apply -f manifests/ -R
+```
+
+The above command sets up JupyterHub, an API for training using Tensorflow, and a set of deployment files for serivng. 
+Used together, these serve as configuration that can help a user go from training to serving using Tensorflow with minimal
+effort in a portable fashion between different environments. You can refer Instructions for using each of these components below.
```
Comment (jlewi) [inline kubeflow/README.md]: "refer to instructions"

**Comment by aronchick** [comment]
I'm happy to take a swing today.

File: kubeflow/README.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+```commandline
+kubectl apply -f manifests/ -R
+```
+
+The above command sets up JupyterHub, an API for training using Tensorflow, and a set of deployment files for serivng. 
+Used together, these serve as configuration that can help a user go from training to serving using Tensorflow with minimal
+effort in a portable fashion between different environments. You can refer Instructions for using each of these components below. 
+
+## Minikube
```
Comment (jlewi) [inline kubeflow/README.md]: Do we need to have top level sections for minikube and GKE which then have the same subections (e.g. single node, distributed training).

The message we want to send is that the commands on minkube are the same as GKE.

So ideally we had have a single set of instructions for the various steps that should work regardless of the deployment.

For training this should work; i.e. to launch a training job the command should be

kubectl create -f tf_job.yaml



/cc @aronchick

Comment (foxish) [reply kubeflow/README.md]: I considered that, and switched between layouts a bit. Open to suggestions.

Comment (foxish) [reply kubeflow/README.md]: Done, changed layout to have less duplication.

**Comment by vishh** [comment]
This is a good start. @aronchick can you do some wordsmithing to get this to a reasonable shape. This should be the Readme in the google/kubeflow repo.

**Comment by vishh** [comment]
@foxish is this still relevant?

**Comment by foxish** [comment]
This can be closed. It's been moved to https://github.com/google/kubeflow

**Pull request closed without merging**